modules/client-configuration/pages/clients-ubuntu.adoc

```diff
@@ -132,13 +145,15 @@ Your custom channels should use this structure, at menu:Software[Channel List >
 * Child custom channel: [systemitem]``Ubuntu 18.04 LTS AMD64 Main Security``
 * Child vendor channel: [systemitem]``Ubuntu-18.04-SUSE-Manager-Tools for amd64``
 
-When you have the channels set up, associate each channel with the appropriate repository, and synchronize then channels (optionally configure scheduled syncronization).
+When you have the channels set up, associate each channel with the appropriate repository, and synchronize the channels.
+You can also configure a synchronization schedule at this point.
 
-To do this, proceed to menu:Software[Manage > Channels], and for each channel you created, click on it and:
+To synchronize the channels, navigate to menu:Software[Manage > Channels], and for each channel you created, click on it and:
```

Comment:
Instead of "click on it" I'd either rather say "click it" or "select it".

Later l.196 we have this variation:

> Click each channel in the list, and: